src/lib/utils/options/options.ts

```diff
@@ -303,6 +303,9 @@ export class Options {
             // No need to convert the defaultValue for a map type as it has to be of a specific type
             if (declaration.type === ParameterType.Map) {
                 this._values[declaration.name] = declaration.defaultValue;
+            } else if (declaration.type === ParameterType.Number) {
+                // Don't use convert for number options to allow every possible number as a default value
+                this._values[declaration.name] = declaration.defaultValue || 0;
```

Comment:
So, "every possible number" doesn't include NaN? I think I'm okay with this... will sleep on it.

Reply:
Yes, I was thinking about that one too. Maybe allowing `NaN` is the better solution though. Otherwise I would wonder why the plugin resets it to `0`.

Reply:
I think disallowing `NaN` is a good idea. It is such a common source of bugs that I'd rather just disallow it entirely. If someone *really* wants an obviously bad value, they can use `-Infinity` or something.

Reply:
@Gerrit0 Yeah, that's probably the better solution for now. That way you can be sure to have "a number" as a value.